net.sourceforge.Lifeograph.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id": "net.sourceforge.Lifeograph",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "lifeograph",
+    "finish-args": [ "--socket=x11", "--socket=wayland", "--filesystem=host" ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": ["/include", "/lib/*/include", "/lib/pkgconfig", "/share/pkgconfig", "/share/aclocal", "/man", "/share/man", "/share/info", "/share/gtk-doc", "*.la", "*.a" ],
```

Comment:
Same as finish-args.

Reply:
addressed in 06866a5dea7ad4fea7401422926a8a6ba70424db